Source/Core/Core/HW/DSPLLE/DSPSymbols.h

```diff
@@ -23,7 +23,6 @@ class DSPSymbolDB : public SymbolDB
 
 extern DSPSymbolDB g_dsp_symbol_db;
```

Comment:
Hmm, maybe it would be useful to find the old annotated assembly files in Git and pull out a list of their symbols and locations?